docs/uv/manager.rst

```diff
@@ -0,0 +1,64 @@
+Measurement Manager
+==========
```

Comment:
Did this render okay? For some reason (and it would be silly) I thought you had to stretch the under-equals-signs all the way to the end of the line.

Reply:
Yep, it renders fine for me